omod/src/main/resources/config.xml

```diff
@@ -111,4 +115,10 @@
 		<defaultValue></defaultValue>
 		<description>Properties file path for procedure template</description>
 	</globalProperty>
+
+	<globalProperty>
+		<property>fhir.export.anonymise.config.path</property>
+		<defaultValue></defaultValue>
+		<description>Properties file path for fhir export anonymisation</description>
```

Comment:
Path of the config file that controls the anonymisation mechanism during bulk-export of patient data.